docs/tasks.md

```diff
@@ -153,3 +139,172 @@ work has completed, spinning up replacement workers.
+
+#### Profile SQL
+
+This task was added as a new feature in Tower 3.5. It allows the user to turn on a global profiler in their system, so that Tower can profile all of the SQL queries that they make.  This is a "fanout" style task (meaning all nodes execute it), and one of the main benefits is that it assists with identifying slow queries.
+
+
+#### Gather Analytics
+
+The analytics collection `gather()` and `ship()` functions are called by an `awx-manage gather_analytics --ship` command, which runs on whichever Tower instance it is invoked on. When these functions are called by Celery beat (currently at midnight local time), it is run on one `execution_node`, or Tower instance, by the Python in the AWX virtualenv.
+
+For more details about analytics, please visit the [Usability Analytics and Data Collection](https://docs.ansible.com/ansible-tower/latest/html/administration/usability_data_collection.html) page.
+
+
+#### Run Administrative Checks
+
+Not applicable to AWX, this task checks that the Tower license currently in use is valid and alerts the admin user(s) via email when they are in danger of going over capacity and/or when the license is about to expire. Specifically (in cases of going over capacity), it triggers when the node count is at or over 90% of what the license allows.
+
+
+#### Purge Old Stdout Files
+
+AWX sometimes buffers `stdout` for playbook runs to disk _when users download stdout for a job run_.  This task implements a periodic cleanup of the directory where this data is stored.
+
+
+#### Delete Project Files
+
+A "fanout" task (meaning all nodes execute it), this looks at the local file system and deletes project-related files (generally, source control clones) when a Project is deleted from AWX.
+
+
+#### Handle Work Success/Error
+
+This task is part of the process of running a unified job.  For example, let's say that a Project Update gets started, and it takes 10 seconds; it's done and ready to go with no more dependencies.  Instead of waiting for the scheduler to wake up again (typically every 30 seconds), this task will alert the scheduler to go ahead and run the next phase of the dependency graph.
+
+In case of an error, the same thing happens as above but with a "fail" vs a "success".  So for example if a workflow node is set to run "on fail", the Handle Work task will wake up the scheduler and ensure that the next step runs.  It is also useful in recording cascading errors (_e.g._, if a job has an error, it will look at what it depended on and report status details for all of those dependencies as well).
+
+
+#### Send Notifications
+
+When a user creates a notification at `/api/v2/notifications/`, they can assign it to any of the various objects that support it (_i.e._, all variants of Job Templates as well as Organizations and Projects).  They can also set the appropriate trigger level for when they want the notification task to run (_e.g._, error, success, or any).
```

Comment:
You can't create things in `/api/v2/notifications/`. You create a notification template in `/api/v2/notification_templates/` and attach it to things. When the thing runs and triggers the notification template, it sends a notification, and the action of sending the notification is recorded in `/api/v2/notifications/`. Notifications are like the unified jobs, but not a unified job.